mindocr/models/det_dbnet.py

```diff
@@ -84,30 +86,32 @@ def dbnet_resnet50(pretrained=False, **kwargs):
     return model
 
 
-'''
 @register_model
-def dbnet_mv3(pretrained=False, **kwargs):
+def dbnetpp_resnet50(pretrained=False, **kwargs):
     model_config = {
-            "backbone": {
-                'name': 'det_mv3',
-                'pretrained': False
-                },
-            "neck": {
-                "name": 'FPN',
-                "out_channels": 128,
-                },
-            "head": {
-                "name": 'ConvHead',
-                "out_channels": 2,
-                "k": 50
-                }
-            }
+        "backbone": {
+            'name': 'det_resnet50',
+            'pretrained': not pretrained
```

Comment:
Although the logic is correct. it is still hard to understand at the first sight due to the same name. 
Suggest:  explicitly define backbone_pretrained = not pretrained, before Line 92, and 'pretrained': backbone_pretrained

Reply:
Improved